src/Backend/Teradata/ToFinalTable/SqlBuilder.php

```diff
@@ -88,7 +107,7 @@ public function getInsertAllIntoTargetTableCommand(
                 $columnsSetSql[] = sprintf(
                     'CAST(COALESCE(%s, \'\') as %s) AS %s',
                     TeradataQuote::quoteSingleIdentifier($columnDefinition->getColumnName()),
-                    $columnDefinition->getColumnDefinition()->buildTypeWithLength(),
+                    $destinationColumn->getColumnDefinition()->buildTypeWithLength(),
```

Comment:
to je divny, to jsem menil ne...

Reply:
Taky sem měl pocit, že sme to řešili a fixovali, ale kód nekecá.